modules/i18n/src/translations/fr-FR/portals/console.ts

```diff
@@ -4423,7 +4423,9 @@ export const console: ConsoleNS = {
                             validationErrorMessages: {
                                 duplicateName: "L'URI de l'attribut {{type}} existe déjà.",
                                 invalidName: "Le nom que vous avez entré contient des caractères non autorisés. " +
-                                    "Seuls les alphabets, les nombres, «#», «_» sont autorisés."
+                                    "Seuls les alphabets, les nombres, «#», «_» sont autorisés.",
```

Comment:
What's the use of `#` here? And why it is wrapped with `<< >>` ?

Reply:
`#` is added for local attribute name validation.